config/locales/en.yml

```diff
@@ -1082,6 +1084,7 @@ en:
           unsure: I am not sure
       steps_lateness_lateness_reason_form:
         heading: Why are you late with your appeal?
+        not_sure_heading: Are there reasons why your appeal might be late?
```

Comment:
You will need to request Welsh translations for this from Jamil, and add them to config/locales/cy.yml . It's a bit of a pain because it causes a delay.

Reply:
On it

Reply:
done